master/buildbot/reporters/mail.py

```diff
@@ -43,6 +43,11 @@
 from buildbot.reporters.message import MessageFormatter as DefaultMessageFormatter
 from buildbot.util import service
 
+if PY3:
```

Comment:
@tardyp I requested clarification of your feedback, and you did not provide it.
Please provide a clear example of what you want, so I can move forward with this patch.

This is a major blocker in terms of moving the buildbot code forward with Python 3.

Thanks.